src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,28 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
```

Comment:
The import for `Logger` should be from `org.apache.logging.log4j.Logger` instead of `org.apache.logging.log4j.core.Logger`. The `core` package is typically used for internal purposes and not for general logging usage.